### ISSUE TITLE
ManyToOne causes weird id name, ignoring JoinColumn's name

### ISSUE THREAD

**Issue by ChristianSch** [TouK/krush#53]
Taking [this test case](https://github.com/TouK/krush/blob/f8a05c9f6b9981ac4d1f9614791241ca5701a2e7/example/src/test/kotlin/pl/touk/krush/one2many/bidi/TreeTest.kt) as an example with [these entities](https://github.com/TouK/krush/blob/de810110697f82fd806099ff6f2c6a3a352bb04e/example/src/main/kotlin/pl/touk/krush/one2many/bidi/Tree.kt), the generated sql looks something like this:

```sql
CREATE TABLE IF NOT EXISTS tree (
    id BIGINT AUTO_INCREMENT PRIMARY KEY,
    `name` VARCHAR(255) NOT NULL);

CREATE TABLE IF NOT EXISTS branch (
    id BIGINT AUTO_INCREMENT PRIMARY KEY,
    `name` VARCHAR(255) NOT NULL,
    treeId_id BIGINT NULL,
    CONSTRAINT fk_branch_treeId_id_id FOREIGN KEY (treeId_id) REFERENCES tree(id) ON DELETE RESTRICT ON UPDATE RESTRICT);

CREATE TABLE IF NOT EXISTS leaf (
    id BIGINT AUTO_INCREMENT PRIMARY KEY,
    `name` VARCHAR(255) NOT NULL,
    branchId_id BIGINT NULL,
    CONSTRAINT fk_leaf_branchId_id_id FOREIGN KEY (branchId_id) REFERENCES branch(id) ON DELETE RESTRICT ON UPDATE RESTRICT);
```

It appears that the `@JoinColumn` is ignored. Compare `tree_id` with `treeId_id`, where I would've expected `tree_id` to be used for the column name. I feel like that is a recent change because it's caused by updating to `0.4.1` and I can't get back because the previous repository seems to be gone. Is this a bug? If not, is there a way to control the name of the join column in this case?

I would much rather not change the migrations

**Comment by Namnodorel** [comment]
I don't know about ignoring `@JoinColumn`,  but the double "id" name should be fixed by something like this commit: https://github.com/TouK/krush/pull/45/commits/bbe9855c9683291e8725a0408163b2ea1c10202b

**Comment by ChristianSch** [comment]
Yeah this looks reasonable. But it doesn't seem to be part of any release yet? The code looks to me as if you really can't control association column names, but the linked example entity hints at the possibility. Not sure what to make of it to be honest.
